testing-guidance/workflow-tests/android/android-lkg-workflows.md

```diff
@@ -0,0 +1,77 @@
+# Android LKG Workflow Tests
+
+Testing in this area should focus on the different ways O3DE can be built and deployed into an android device and the launch of a default level successfully.
+
+## General Android Docs
+* [O3DE User Guide: Android Support for O3DE](https://www.o3de.org/docs/user-guide/platforms/android/)
+
+## Common Issues to Watch For
+
+- Binary and assets are located at their correct place for each packaging mode.
+- The most common packaging mode used during Android development is VFS (Virtual File System), be aware of network connection errors from the mobile device to Asset Processor running on PC.
+- Rendering artifacts when running default level on Android devices.
+- Low framerates when running default level on Android devices. Look out for spamming error/warnings in the Android log.
+- Asserts and Errors that appear in the Android Log while running default level.
+- Immediate crashes when launching the app on the Android device.
+- Crashes that occur after some time leaving the app running.
+
+## Workflows
+
+### Area: Build android in different packaging modes
+
+**Project Requirements**
+* Create a new default project (`DefaultProject`)
+* Modify `o3de\Registry\AssetProcessorPlatformConfig.setreg` to generate android assets in the cache. Remove `//` the following android line:
+````
+"android": "enabled",
+````
+* Make sure `DefaultProject\Platform\Android\androd_project.json` exists and that it has android settings section in it. Example:
```

Comment:
I posit that we use forward slashes for paths in this document since Android is available for use on POSIX platforms such as Linux and IOS.
'
Also there is a typo in the `android_project.json` file.

Finally I would suggest using a placeholder for specifying paths to project root or engine root.
Such as `<project-root>` for the project root.


```suggestion
* Modify `<project-path>/Registry/AssetProcessorPlatformConfig.setreg` to generate android assets in the cache. Remove `//` the following android line:
\````
"android": "enabled",
\````
* Make sure `<project-path>/Platform/Android/android_project.json` exists and that it has android settings section in it. Example:
```

Reply:
Good calls, I'll do the changes. Thanks.